CHANGELOG.md

```diff
@@ -278,132 +338,198 @@ This is the easiest way to work with the released versions of PWA Studio.
 
 #### Upgrade method: Update `package.json`
 
-To upgrade to 4.0.0, update the project's `package.json` file and change the dependency version for PWA Studio.
+To upgrade to 5.0.0, update the project's `package.json` file and change the dependency version for PWA Studio.
+
+## Known issues
```

Comment:
Do we not have more Known Issues to call out? @jimbo @dpatil-magento  please weigh in.

Reply:
@jcalcaben  There is one more, might not specific to 5.0 but visible on local development, where images does not load unless you set NODE_TLS_REJECT_UNAUTHORIZED , #2005 would be solving this issue in future.

Reply:
@dpatil-magento Added in bf35d48